src/plot/axis/axis-title.js

```diff
@@ -23,6 +23,32 @@ import React from 'react';
 import {ORIENTATION} from 'utils/axis-utils';
 
 const {LEFT, RIGHT, TOP, BOTTOM} = ORIENTATION;
+const transformation = (width, height) => ({
+  [LEFT]: {
+    x: 16,
```

Comment:
I hear that, i think we will most likely run into some collision (eg users trying to manipulate distances using em) later around this kind of hard constant setting, but i this is alright for now. What i am suggesting with this nit is that you pulled these numbers up to the top of the file as constants, so it is explicitly clear why they are set this way